src/decisionengine/framework/dataspace/dataspace.py

```diff
@@ -70,24 +76,22 @@ def __init__(self, config):
         :arg config: Configuration dictionary
         """
 
-        self.logger = structlog.getLogger(LOGGERNAME)
-        self.logger = self.logger.bind(module=__name__.split(".")[-1])
-        self.logger.debug('Initializing a dataspace')
+        logger.debug('Initializing a dataspace')
```

Comment:
Using the module logger because there is no need to pass it around to code in other modules?

Reply:
That was my understanding.  The `dataspace.py` file has items inside and outside class structures.  Avoiding duplicate setup calls seemed like a workable approach to me.

Reply:
@mambelli Are you asking why this is logging to "decisionengine" logger and not channel logger?

Reply:
I think the logger is the same (`getLogger(LOGGERNAME)`) so no decisionengine vs channel logger. Before it was assigned to the attribute, now it is a module level variable. In this module it makes no difference since LOGGERNAME is static, so it makes sense. DataSpace has no children, I did not  check if self.logger is used by someone using a DataSpace  instance. Probably not.

Reply:
Yes, I moved it because it is used in the functions in the module, as well as in the Dataspace class. The datablock uses an instance of a dataspace, but does not call the dataspace logger.